### ISSUE TITLE
Opening a private session while in fullscreen mode no longer opens a new tab

### ISSUE THREAD

**Issue by maxime-michel** [qutebrowser/qutebrowser#5549]
**Version info**:
qutebrowser v1.12.0
Git commit: eaff6e82d (2020-06-01 15:57:24 +0200)
Backend: QtWebEngine (Chromium 80.0.3987.163)
Qt: 5.15.0

**Does the bug happen if you start with `--temp-basedir`?**: Yes

**Description**
On OS X, I've always run one single window of qutebrowser, in which I was able to have, first, my regular session. Once in fullscreen mode, I could then open a private session that would stay within the same window, as a separate tab. That wasn't a qutebrowser tab, but a tab above that. Since 1.12 (and 1.13 doesn't help), this opens a new qutebrowser window in another OS X space. Those are impossible to navigate using Alt-Tab. Is there a chance this behavior I was relying on will be coming back?

**How to reproduce**
1. open qutebrowser
2. put it in fullscreen mode
3. `:open -p site.com`
4. a new OS X space is created
5. Alt-Tab becomes a nightmare

**Comment by The-Compiler** [comment]
This functionality isn't coming from qutebrowser - it always opens a new *window* with `:open -p`, so it must be something in macOS (or Qt) grouping them (like you say, *That wasn't a qutebrowser tab, but a tab above that.*).

I don't think there's anything that changed in qutebrowser about this either, it's more likely to be something in the Qt 5.15 update done in qutebrowser v1.12.0 which causes this to happen. Can you confirm the behavior you describe comes back when you go back to v1.11.0? Does the same thing happen with `:open -w` as well? If you happen to re-test with v1.11.0, could you take a screenshot?

**Comment by maxime-michel** [comment]
Thanks for the help, I was afraid this had to do with Qt and is therefore out of your control. This is what it used to look like. `:open -w` yields the same result.

<img width="1280" alt="tabs-of-tabs" src="https://user-images.githubusercontent.com/14178626/85838310-aa1b0c00-b799-11ea-8bb3-11585b902ce3.png">

**Comment by The-Compiler** [comment]
That's a quite nice feature, though! I had a quick look through the Qt bugtracker, and didn't find anything relevant. Maybe someone who knows more about macOS has an idea what's going on? @koekeishiya or @rien333 perhaps?

**Comment by rien333** [comment]
Nope, sorry, I don't even remember using this feature in macOS. Which macOS version are you on? (I haven't really used it in almost two years).

Aside from delving deep into the Qt side of things, you could try altering your work flow, I guess.  iirc, qutebrowser can fake fullscreen (even on macOS), which would allow you to have to have two windows on the same space you could then cycle between. Of course, this might not fit your preferences in some way.

**Comment by maxime-michel** [comment]
I am running 10.15.5, which is latest, and have been on 10.15.x since March, way before 1.12 (June 1st) came out. So it would have to be Qt. Well, I'll deal with it. Should I leave this issue open?

**Comment by The-Compiler** [comment]
@maxime-michel It would need to be an example which actually shows multiple windows in full-screen. Not sure if anything suitable exists inside the Qt examples.
